EvotoApi/Areas/Management/Controllers/RegistrarConnectionController.cs

```diff
@@ -128,5 +128,42 @@ public class RegistrarConnectionController : ApiController
                 });
             }
         }
+
+        [Route("settings/list")]
+        //[Authorize]
+        [HttpGet]
+        public async Task<IHttpActionResult> ListSettings()
+        {
+            try
+            {
+                var settings = await RegistrarConnection.ListRegistrarSettings();
+                return Json(settings);
+            }
+            catch (Exception e)
+            {
+                ModelState.AddModelError("error", e);
```

Comment:
should be e.Message? Regardless, shouldn't be revealing internal errors, and should probably just return internalservererror